### PR TITLE
Issue4

### LINKED ISSUE #4: Implement Data Extraction Operation for current season schedule, current week schedule, season schedule of Game Feeds #4
**Description:**
As part of our ongoing efforts to enhance the functionality of the GameFeeds service, there's a need to implement a data extract operation for improving the handling of current season schedule, current week schedule, season schedule.

**Tasks:**

- [x] Implement a new function for extracting the raw current season schedule data into a structured format that's easy to process.
- [x] Implement a similar function for the current week schedule data. 
- [x] Develop a function for dealing with the season schedule data. 
- [x] Write appropriate unit tests for the new functionality to ensure reliability and ease of integration.

**Acceptance Criteria:**

- [x] The raw data from the GameFeeds service should be successfully extracted.
- [x] The transformation functions should have complete unit test coverage and should pass all tests.
- [x] Ensure that the changes do not break any existing functionality of the GameFeeds service.

### PR DESCRIPTION
**Pull request by jbassie**
Created the following functions for games_feed

- get_current_season_schedule
- get_current_week_schedule
- get_season_schedule
- get_week_schedule

Also Implemented Unit Tests for each of the functions

### CODE REVIEW

File: src/test/unit/sportsradar/extract/gamefeeds.py [hunk 1]
```diff
@@ -54,3 +54,54 @@ def test_game_feeds_with_game_statistics(mock_data_store, mock_getenv):
                                         game_id=game_id, file_format=TestConstants.FORMAT,
                                         api_key=TestConstants.API_KEY)
     assert result.status_code == 200
+
```
Comment (dataengineergaurav) [inline src/test/unit/sportsradar/extract/gamefeeds.py]: please apply new unit tests updated on the main branch.

**Review by dataengineergaurav — CHANGES_REQUESTED** [review]
Please do a fresh pull from the main branch and apply relevant unit tests for the this.

File: src/test/unit/sportsradar/extract/gamefeeds.py (hunk 1 above)
Comment (jbassie) [reply src/test/unit/sportsradar/extract/gamefeeds.py]: Changes Implemented

**Comment by dataengineergaurav** [comment]
I have checked unit tests. It successfully passes the tests but has some None values. 
Any ideas?
image

**Review by dataengineergaurav — APPROVED** [review]

**Pull request merged**